tree/ntuple/v7/test/ntuple_basics.cxx

```diff
@@ -701,3 +701,45 @@ TEST(RNTuple, ReadCallback)
    EXPECT_EQ(1337.0, rdKlass->a);
    EXPECT_EQ(6U, gNCallReadCallback);
 }
+
+TEST(RNTuple, FillBytesWritten)
+{
+   FileRaii fileGuard("test_ntuple_fillbytes.ntuple");
+
+   // TODO(jalopezg): improve test coverage by adding other field types
+   {
+      auto model = RNTupleModel::Create();
+      auto fieldI32 = model->MakeField<std::int32_t>("i32");
+      auto fieldStr = model->MakeField<std::string>("s");
+      auto fieldBoolVec = model->MakeField<std::vector<bool>>("bool_vec");
+      auto fieldFloatVec = model->MakeField<std::vector<float>>("float_vec");
+      auto ntuple = RNTupleWriter::Recreate(std::move(model), "f", fileGuard.GetPath());
+      *fieldI32 = 42;
+      *fieldStr = "abc";
+      // A 32bit integer + "abc" literal + one 64bit integer for each index column
+      EXPECT_EQ(31U, ntuple->Fill());
+      *fieldBoolVec = {true, false, true};
+      *fieldFloatVec = {42.0f, 1.1f};
+      // A 32bit integer + "abc" literal + one 64bit integer for each index column + 3 bools + 2 floats
+      EXPECT_EQ(42U, ntuple->Fill());
+   }
+
+   {
```

Comment:
The 2 fragments looks identical as first glance.  Please highlight the differences and/or consider using a single function covering both.